TabLayout/tests/TabGroup-specs.js

```diff
@@ -87,61 +96,8 @@ describe('TabGroup specs', () => {
 				]}
 			/>
 		);
+		const iconList = screen.getByTestId('tabGroup').children.item(0);
 
-		const expected = false;
-		const actual = subject.find('Tab').prop('disabled');
-
-		expect(actual).toBe(expected);
-	});
-
-	it('should generate keys automatically', () => {
-		const subject = mount(
-			<TabGroup
-				tabs={[
-					{title: 'Home', icon: 'home'},
-					{title: 'Button', icon: 'demosync'},
-					{title: 'Item', icon: 'playcircle'}
-				]}
-			/>
-		);
-
-		const expected = 'tabs_Homehome';
-		const actual = subject.find('GroupItem').first().key();
-
-		expect(actual).toEqual(expected);
-	});
-
-	it('should use a custom key if supplied', () => {
-		const subject = mount(
-			<TabGroup
-				tabs={[
-					{title: 'Home', icon: 'home', tabKey: 'myCustomKey'},
-					{title: 'Button', icon: 'demosync'},
-					{title: 'Item', icon: 'playcircle'}
-				]}
-			/>
-		);
-
-		const expected = 'myCustomKey';
-		const actual = subject.find('GroupItem').first().key();
-
-		expect(actual).toEqual(expected);
-	});
-
-	it('should support zero as a custom key', () => {
-		const subject = mount(
-			<TabGroup
-				tabs={[
-					{title: 'Home', icon: 'home', tabKey: 0},
-					{title: 'Button', icon: 'demosync'},
-					{title: 'Item', icon: 'playcircle'}
-				]}
-			/>
-		);
-
-		const expected = '0';
-		const actual = subject.find('GroupItem').first().key();
-
-		expect(actual).toEqual(expected);
```

Comment:
Note to reviewers: Last 3 tests refers to implementation details of the `TabGroup` component, I couldn't find a way to migrate them to react-testing-library

Reply:
I agree with you. It's not what the user would see.